apps/frontend/src/components/ui/moderation/ModerationTechRevCard.vue

```diff
@@ -0,0 +1,1107 @@
+<script setup lang="ts">
```

Comment:
A lot of the of the logic in this component could probably use the DI framework as well